skypy/galaxy/spectrum.py

```diff
@@ -197,6 +198,77 @@ def mag_ab(spectrum, bandpass, redshift=None):
     return mag_ab.item() if not return_shape else mag_ab.reshape(return_shape)
 
 
+def magnitudes_from_templates(coefficients, templates, bandpasses, redshift=None,
+                              resolution=1000, stellar_mass=None, distance_modulus=None):
```

Comment:
Could this be `bandpass` just for consistency with `mag_ab`? I know this is quite an annoying request for all the follow-up changes below so feel free to disregard.